### PR TITLE
GitHub action latest tag

### PR DESCRIPTION
**Pull request by Alkarex**
Automatically tag the latest release with the `latest` git tag
https://github.com/marketplace/actions/latest-tag

### CODE REVIEW

**Comment by Alkarex** [comment]
P.S. I am not entirely sure of the syntax :-P

File: .github/workflows/latest.yml [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+  release:
+    types: [published]
+
+jobs:
+  run:
+    name: Run local action
+    runs-on: ubuntu-latest
+    steps:
+      - name: Run latest-tag
+        uses: EndBug/latest-tag@6d22a6738f5c33059e3a8c6ca5dcf8eaf8a14599
```
Comment (Frenzie) [inline .github/workflows/latest.yml:12]: They don't have a version?

Comment (Alkarex) [reply .github/workflows/latest.yml:12]: Yes they have, but this one is more precise (like when using a library with version pinning) - I am not entirely sure of what those marketplace actions can do if they are updated to something evil :-)

**Comment by Alkarex** [comment]
Let's merge it, so we can see whether it works for 1.18.1

**Pull request merged**